src/Resources/app/administration/src/component/adyen-refund/adyen-refund.html.twig

```diff
@@ -0,0 +1,47 @@
+{% block adyen_refund %}
+    <sw-card :hero="false" :isLoading="isLoadingTable" :large="false" title="Refunds" v-if="showWidget">
+        <div v-if="!errorOccurred">
+            <div class="adyen-card-header">
+                <div class="adyen-card-header-first">
+                    <span v-if="!refunds.length">{{ $tc('adyen.noRefundsCreated') }}</span>
+                </div>
+                <div id="modalButtonContainer">
+                    <sw-button
+                        :disabled="!allowRefund"
+                        :square="false"
+                        :block="false"
+                        @click="openModal"
+                        class="sw-button--small">
+                        {{ $tc('sw-order.documentCard.labelCreateNew') }}
+                    </sw-button>
+                </div>
+                <sw-modal v-if="showModal"
+                        @modal-close="onCloseModal"
+                        :title="$tc('adyen.createRefund')"
+                        variant="small">
+                    {{ $tc('adyen.refundConfirm') }} <b>{{ order.orderNumber }}</b>?
```

Comment:
I would remove the question mark so that the translation is easier. Something like this
```suggestion
                    {{ $tc('adyen.refundConfirm') }}: <b>{{ order.orderNumber }}</b>
```